code/datums/mapgen/Cavegens/IcemoonCaves.dm

```diff
@@ -8,7 +8,7 @@
 		/mob/living/basic/mining/lobstrosity = 15,
 		/mob/living/basic/mining/goldgrub = 10,
 		/mob/living/simple_animal/hostile/asteroid/polarbear = 30,
-		/mob/living/simple_animal/hostile/asteroid/wolf = 50,
+		/mob/living/basic/mining/wolf = 50,
```

Comment:
this list should be alphabetized. there's a few extensions for VSC that will automatically do this for you.

you _could_ also sort it by weight but it wasn't like that beforehand so just alphabetical is fine